Source/WebCore/rendering/svg/LegacyRenderSVGRoot.cpp

```diff
@@ -69,6 +69,14 @@ SVGSVGElement& LegacyRenderSVGRoot::svgSVGElement() const
     return downcast<SVGSVGElement>(nodeForNonAnonymous());
 }
 
+bool LegacyRenderSVGRoot::hasIntrinsicAspectRatio() const
+{
+    FloatSize intrinsicSize;
+    double intrinsicRatio = 0;
+    computeIntrinsicRatioInformation(intrinsicSize, intrinsicRatio);
```

Comment:
Since I moved the logic of computing the transferred min/max sizes and constraining the computed size into `computeReplacedLogicalWidthRespectingMinMaxWidth` (and height), we don't need the second method to constrain anymore. We just need `computeIntrinsicRatioInformation` to get the intrinsic ratio and sizes. All of the constraining will then be done in the other method.

Reply:
This is no longer true since the change was causing too many issues. computeReplacedLogicalWidthRespectingMinMaxWidth is only constraining with definite sizes in the same dimension. renderreplaced::computeIntrinsicSizesConstrainedByTransferredMinMaxSizes is constraining using the transferred min/max sizes.